features/payment/managing_payment_methods/adding_payment_method.feature

```diff
@@ -8,22 +8,28 @@ Feature: Adding a new payment method
         Given the store is available in "English (United States)"
         And I am logged in as an administrator
 
-    @ui
+    @ui @javascript @current
     Scenario: Adding a new payment method
         Given I want to create a new payment method
         When I name it "Paypal Express Checkout" in "English (United States)"
         And I specify its code as "PEC"
         And I choose "Paypal Express Checkout" gateway
+        And I specify its "Username" as "test"
```

Comment:
It is, of course, but in context there should be a specific method for any field.